server/routes/assessment.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+
+export default function createRouter(assessment) {
+  const router = express.Router();
+
+  router.post('/', (req, res) => {
+    assessment.record(req.body)
+      .then(
+        result => res.json({
+          status: 'OK',
+          data: { id: result.assessment_id },
+        }),
+        (err) => {
+          // TODO: proper logging lib for server stuff?
```

Comment:
I think we should re-work logging in a separate ticket, if I add that jira does it need to go somewhere so it doesn't get lost?